src/app/game/memory-flip-card/page.tsx

```
@@ -0,0 +1,351 @@
+        <p className={styles.subtitle}>Challenge your mind and improve your memory!</p>
+      </div>
+
+      {/* Difficulty Selection */}
+      <div className={styles.difficultySelector}>
+        {(['easy', 'medium', 'hard'] as const).map((level) => (
+          <button
+            key={level}
+            className={`${styles.difficultyButton} ${difficulty === level ? styles.active : ''}`}
+            onClick={() => setDifficulty(level)}
+          >
+            {level.charAt(0).toUpperCase() + level.slice(1)} ({level === 'easy' ? '6' : level === 'medium' ? '8' : '12'} pairs)
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)